src/analysis/normed_space/star/spectrum.lean

```diff
@@ -62,3 +66,130 @@ lemma self_adjoint.coe_spectral_radius_eq_nnnorm (a : self_adjoint A) :
 spectral_radius_eq_nnnorm_of_self_adjoint a.property
 
 end complex_scalars
+
+/-- The inclusion of the base field in a algebra as a continuous linear map. -/
+@[simps]
+def algebra_map_clm (𝕜 : Type*) (E : Type*) [normed_field 𝕜] [semi_normed_ring E]
+  [normed_algebra 𝕜 E] : 𝕜 →L[𝕜] E :=
+{ to_fun := algebra_map 𝕜 E,
+  map_add' := (algebra_map 𝕜 E).map_add,
+  map_smul' := λ r x, by rw [algebra.id.smul_eq_mul, map_mul, ring_hom.id_apply, algebra.smul_def],
+  cont := (algebra_map_isometry 𝕜 E).continuous }
+
+lemma algebra_map_clm_coe (𝕜 : Type*) (E : Type*) [normed_field 𝕜] [semi_normed_ring E]
+  [normed_algebra 𝕜 E] : (algebra_map_clm 𝕜 E : 𝕜 → E) = (algebra_map 𝕜 E : 𝕜 → E) := rfl
```

Comment:
It would be nice to also add that `(algebra_map_clm 𝕜 E).to_linear_map = algebra.linear_map 𝕜 E`